pages/1.10/security/ent/perms-reference/index.md

```diff
@@ -49,6 +49,7 @@ By convention, `full` indicates that the permission supports all other action id
 | `dcos:adminrouter:ops:system-logs` <br>Controls access to [System logs API](/1.10/api/master-routes/#system).                                                                                                                                                                      | x    |   |   |   |   |
 | `dcos:adminrouter:ops:system-metrics`<br> Controls access to [System metrics API](/1.10/api/master-routes/#system).                                                                                                                                                                | x    |   |   |   |   |
 | `dcos:adminrouter:package` <br> Controls access to the [Cosmos API](/1.10/api/master-routes/#cosmos), which provides access to the DC/OS Universe.                                                                                                                                 | x    |   |   |   |   |
+| `dcos:adminrouter:secrets` <br>  Controls access to [Secrets UI](/1.10/gui/secrets/).                                                                                                                                                                      | x    |   |   |   |   |
```

Comment:
"to *the* Secrets UI"

applies in multiple places

Reply:
"to the Secrets API"!

This is _much_ more about the API itself (the HTTP flow), than the UI.

Reply:
As I've confirmed in https://jira.mesosphere.com/secure/EditComment!default.jspa?id=79240&commentId=139792 this permission doesn't affect access to API in general, but rather only controls access to secrets UI.

Reply:
@mhrabovcin Admin Router indeed does not do authorization before proxying HTTP requests to the secrets service. It only requires authenticated HTTP requests:

https://github.com/mesosphere/dcos-enterprise/blob/7480c517de78b893917bb9710be2c30327a6d2f6/packages/adminrouter/extra/src/lib/auth/ee.lua#L357

Then this is an even more annoying bug in the UI's "authorization logic" than we thought.